code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -41,7 +42,7 @@
 	permeability_coefficient = 1
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	resistance_flags = FLAMMABLE
-	dog_fashion = /datum/dog_fashion/head/blue_wizard
```

Comment:
removing ian's wizard hat?

Reply:
It's the same value of `/obj/item/clothing/head/wizard/var/dog_fashion`